info.json

```diff
@@ -3,6 +3,7 @@
   "appBase": "https://app.wire.com",
   "bundleId": "com.wearezeta.zclient.mac",
   "copyright": "© Wire Swiss GmbH",
+  "customProtocolName": "wire",
```

Comment:
We can convert `info.json` to `info.js` and replace `"wire"` with `process.env.APP_CUSTOM_PROTOCOL_NAME || 'wire'`.

Reply:
This would need to go in another PR.

Reply:
I agree.